### ISSUE TITLE
Following a guide from packages/contracts/README.md, all contracts are deployed to L1

### ISSUE THREAD

**Issue by vbaranov** [ethereum-optimism/optimism#2083]
I am trying to make a new Optimism deployment. I started with the deployment of contracts. Following the guide from packages/contracts/README.md, the whole set of contracts is deployed to the L1 network as a result, whereas, I suppose, that contracts prefixed with `OVM_` should be deployed to L2. I tried to find mode detailed explanation of each contract, but the related link is broken https://github.com/ethereum-optimism/optimism/issues/2082. Is any straightforward guide how to make a new Optimism deployment to the chain, where it doesn't exist yet: contracts deployment + data transport layer + L2 node?

**Comment by smartcontracts** [comment]
I agree that we should include a guide for deploying the entire system. Some of our documentation for deploying the contracts is very out of date.

**Comment by smartcontracts** [comment]
We've updated the contracts README. I think this issue is better suited for the community-hub which contains all of the docs for Optimism.
